src/app/utils.css

```diff
@@ -1,3 +1,13 @@
 .card-grid-cols-3 {
     grid-template-columns: repeat(auto-fill, minmax(min(400px, 100%), 3fr));
+}
+
+.card-grid-cols-5 {
+    grid-template-columns: repeat(auto-fill, minmax(min(200px, 100%), 5fr));
+}
+
+.favorites-page {
+    overflow: scroll;
+    scrollbar-color: grey;
+    scrollbar-width: thin;
```

Comment:
Hmmm, interesting rule. Does it look ugly without it?

Reply:
Yes. Without this rule, if I have a big number of products to favorites, the whole page scrolls. Also, because there is a certain width to the section, the products are "overflowing"

Reply:
I was talking about the `thin` rule. The scroll is handled by `overflow: scroll`